### ISSUE TITLE
regarding training set for using your model

### ISSUE THREAD

**Issue by wenouyang** [DrSleep/tensorflow-deeplab-resnet#88]
Hi Vladimir,

Thanks for sharing the code. My data set has about 150 image pairs, each of which has a resolution of 500*700.  The mask image has only two classes, background and foreground. Across the image set, the foreground class only takes about 25% of whole image area. An example image is shown as attached.

Since the training image set has only about 150 pairs, I am planning to randomly sample small patches from each image to construct new training set. Generally, what are the best sampling strategy in accordance with the background/foreground ratio? How many patches you suggested for training your network model. 

Thanks,

![capture](https://user-images.githubusercontent.com/5430158/27058339-3d0b29d8-4f96-11e7-9807-907962dc0051.JPG)

**Comment by DrSleep** [comment]
Unfortunately, I cannot give a generic answer to your question. I would say that ideally you would want to have both classes presented in each image in your batch. As for the proportion of the background/foreground in each image, I am unable to comment much (of course, 50-50 balance would seem to work the best a priori, but you would need to evaluate it empirically).
